mono/metadata/mono-basic-block.c

```diff
@@ -336,6 +336,8 @@ bb_formation_il_pass (const unsigned char *start, const unsigned char *end, Mono
 	MonoSimpleBasicBlock *branch, *next, *current;
 	const MonoOpcode *opcode;
 
+	// Note: mono_error_init is not called, assume mono_basic_block_split did so
+
```

Comment:
OK. Even though this is effectively a helper function for `mono_basic_block_split`?